src/policy/concrete.rs

```diff
@@ -128,6 +127,30 @@ impl fmt::Display for PolicyError {
 }
 
 impl<Pk: MiniscriptKey> Policy<Pk> {
+    /// Single-Node compilation
+    #[cfg(feature = "compiler")]
+    fn compile_leaf_taptree(&self) -> Result<TapTree<Pk>, Error> {
+        let compilation = self.compile::<Tap>().unwrap();
+        Ok(TapTree::Leaf(Arc::new(compilation)))
+    }
+
+    /// Extract the Taproot internal_key from policy tree.
+    #[cfg(feature = "compiler")]
+    fn extract_key(&self, unspendable_key: Option<Pk>) -> Result<(Pk, &Policy<Pk>), Error> {
```

Comment:
In the first commit 6fb1b29a46ad10231f50f490300ee22a0a66fc07: Is extract key not completely implemented?

Reply:
Right, this might seem confusing. Will remove it altogether in the first commit.

Reply:
Resolved in 2b136940